docker-compose.yml

```diff
@@ -1,18 +1,24 @@
-version: "2.0"
+version: '3'
```

Comment:
use docker-compose v3

Reply:
this might be a breaking change on environments which don't support docker compose v3, I am in favour of going for it but we have to release it with the next major version and announce it as a potential breaking change.

Reply:
ok so putting this PR on pending until then?